Components/BasketHelper.php

```diff
@@ -466,8 +500,6 @@ public function recalculate(CheckResult $checkResult)
         $this->basket['sShippingcostsNet'] += $shippingCostsNet;
         $this->basket['sShippingcostsWithTax'] += $shippingCostsWithTax;
 
-        // Update total amount
-        $this->basket['AmountNumeric'] += $shippingCostsNet;
```

Comment:
isn't `$this->basket['AmountNumeric'] += $shippingCostsWithTax;` right and neccessary?

Reply:
Some lines below at the end of this method I saw ```$this->basket['AmountNumeric'] = $this->basket['sAmount'];```. This is the reason why I deleted this line.